ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_telemetry_postcard.schema.yml

```diff
@@ -0,0 +1,103 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_telemetry_postcard:
+    type: dict
+    keys:
+      disabled:
+        type: bool
+        description: Enable or disable the postcard telemetry feature
+      ingress:
+        type: dict
+        description: Configure ingress parameters
+        keys:
+          collection:
+            type: dict
+            description: Collector configuration
+            keys:
+              source:
+                type: str
+                description: GRE source configuration
+              destination:
+                type: str
+                description: GRE destination configuration
+              version:
+                type: int
+                description: Postcard version
+                valid_values: [ 1, 2 ]
+          sample:
+            type: dict
+            description: Configure sampling parameters
+            keys:
+              rate:
+                type: int
+                description: Configure sampling rate
+                convert_types:
+                - str
+                valid_values: [ 16384, 32768, 65536 ]
+              tcp_udp_checksum:
+                type: int
+                description: Configure TCP/UDP checksum or IP ID checksum
+                convert_types:
+                - str
+      profiles:
+        type: list
+        description: Configure the postcard telemetry profile
+        primary_key: name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Profile name
+            ingress_sample_policy:
+              type: str
+              description: Configure ingress parameters
+      sample_policies:
+        type: list
+        primary_key: name
+        description: Configure sample policies
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              description: Configure sample policy name
+            match_rules:
+              type: list
+              primary_key: rule_name
+              description: Configure match rules
+              items:
+                type: dict
+                keys:
+                  rule_name:
+                    type: str
+                    description: Match rule name
+                  type:
+                    type: str
+                    description: Select the IP version
+                    valid_values: [ ipv4, ipv6 ]
+                  destination_prefix:
+                    type: str
+                    description: IPv4 Network/Mask or IPv6 Network/Mask
+                  source_prefix:
+                    type: str
+                    description: IPv4 Network/Mask or IPv6 Network/Mask
+                  protocol:
```

Comment:
I think you can flatten this a bit by removing this dict, and just have destination_port and `protocol: <tcp|udp>` in the dict together with destination_prefix etc.

Reply:
Fixed